### DISCUSSION TITLE
Extra Pip Arguments didn't worked as shown on documentation.

### DISCUSSION THREAD

**Discussion by aiglesiasn** [UnitedIncome/serverless-python-requirements#776] [Q&A]
Hi!

I was trying to add a couple of pip arguments but using the documentation example didn't work. This didn't work:
```
pipCmdExtraArgs:
      - --platform manylinux2014_aarch64
      - '--only-binary=:all:'
```
```
pipCmdExtraArgs:
      - '--platform manylinux2014_aarch64'
      - --only-binary=:all:
```
```
pipCmdExtraArgs:
      - '--platform manylinux2014_aarch64'
      - '--only-binary=:all:'
```

I was getting an error on docker that none of those options were valid.
I'm on a Intel Mac mini and I tried it with homebrew python3.10 and also venv python3.10. I even try it inside a docker container with `ubuntu` image, but didn't work.

What end up working was using this:
```
pipCmdExtraArgs: ['--platform', 'manylinux2014_aarch64', '--only-binary=:all:']
```

Has anyone else have since this? Should documentation be changed?

I'm on:
```
node v14.20.1
npm 6.14.18
sls:
Framework Core: 3.31.0
Plugin: 6.2.3
SDK: 4.3.2

```

**Comment by tmlmbrt** [comment]
It's actually 3 cmd line args to pip, right?.  So did you try the following?
- '--platform' 
- 'manylinux2014_aarch64'
- '--only-binary=:all:'
